docs/client/session.md

```diff
@@ -0,0 +1,33 @@
+# Semantic conventions for session
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+Session is defined as the period of time encompassing all activities performed by the application and the actions
+executed by the end user. The period of time is defined by the initial activity and concluded after a period of
+inactivity, or explicitly ended after a maximum duration is reached.
+
+Consequently, a Session is represented as a collection of Logs, Events, and Spans emitted by the Client Application
+throughout the Session's duration. Each Session is assigned a unique identifier, which is included as an attribute in
+the Logs, Events, and Spans generated during the Session's lifecycle.
+
+## Attributes
+
+<!-- semconv client-session-id -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `client.session_id` | string | A GUID. | `00112233-4455-6677-8899-aabbccddeeff` | Opt-In |
```

Comment:
```suggestion
| `client.session_id` | string | A UUID. | `00112233-4455-6677-8899-aabbccddeeff` | Opt-In |
```

As far as I understand it UUID is the more common name and also used more consistently throughout the existing semantic conventions. [For reference](https://datatracker.ietf.org/doc/html/rfc4122):

> This specification defines a Uniform Resource Name namespace for
   UUIDs (Universally Unique IDentifier), also known as GUIDs (Globally
   Unique IDentifier).

Is there a need to be more specific around the version of UUID being used?